apidoc/validate.py

```diff
@@ -1,623 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I would just change some docs with typical errors and check both scripts, no big thing to do. I can give you at least a list of differences by tomorrow if it helps. It's only about small things like disallowed the `description` key in function parameters as I remember. The actual docs build script would ignore them anyway.